app/services/albums.js

```diff
@@ -1,13 +1,11 @@
 const rp = require('request-promise');
 const errors = require('../errors');
 const config = require('../../config/index');
+const models = require('../models');
 
 const base_uri = config.common.api.jsonPlaceholderBaseUri;
 
-const options = {
-  headers: { 'User-Agent': 'Request-Promise' },
-  json: 'true'
```

Comment:
why did you delete this?

Reply:
External API doesn't like these options when I do tests in Postman, I had to remove them to get them to work for the external API's `/albums/:id` endpoint